DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -236,6 +236,7 @@
   </ioread>
 
   <class name="pat::Jet"  ClassVersion="16">
+   <field name="daughtersTemp_" transient="true"/>
```

Comment:
please add the following around L250:
```xml
  <ioread sourceClass = "pat::Jet" version="[1-]" targetClass="pat::Jet" source="" target="caloTowersTemp_">
    <![CDATA[caloTowersTemp_.reset();]]>
  </ioread>
  <ioread sourceClass = "pat::Jet" version="[1-]" targetClass="pat::Jet" source="" target="pfCandidatesTemp_">
    <![CDATA[pfCandidatesTemp_.reset();]]>
  </ioread>
  <ioread sourceClass = "pat::Jet" version="[1-]" targetClass="pat::Jet" source="" target="daughtersTemp_">
    <![CDATA[daughtersTemp_.reset();]]>
  </ioread>
```

this appropriately resets the cache pointers when pat::Jets are loaded